Decred Wallet/Features/Custom Dialogs/AccountSelectorDialog/AccountSelectorDialog.swift

```diff
@@ -0,0 +1,123 @@
+        self.walletsTableView.tableFooterView = UIView(frame: CGRect(x: 0, y: 0, width: 1, height: 0.1))
+        self.walletsTableView.registerCellNib(AccountSelectorTableViewCell.self)
+
+        let accountsFilterFn: (DcrlibwalletAccount) -> Bool = { $0.totalBalance > 0 || $0.name != "imported" }
+        self.wallets = WalletLoader.shared.wallets.map({ Wallet.init($0, accountsFilterFn: accountsFilterFn) })
+        self.walletsTableView.reloadData()
+        
+        let tableContentHeight = self.wallets.reduce(0) { cummulativeContentHeight, wallet in
+            return cummulativeContentHeight + self.walletHeaderSectionHeight
+                + CGFloat(wallet.accounts.count) * self.accountCellRowHeight
+        }
+        self.walletsTableViewHeightConstraint.constant = min(
+            tableContentHeight,
+            UIScreen.main.bounds.height * 0.33 // max height = 1/3rd of screen height
+        )
+    }
+
+    @IBAction func closeButtonTapped(_ sender: Any) {
+        self.dismissView()
+    }
+}
+
+extension AccountSelectorDialog: UITableViewDataSource, UITableViewDelegate {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return self.wallets.count
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return self.walletHeaderSectionHeight
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let label = Label()
+        label.frame = CGRect.init(x: 0, y: 0, width: tableView.frame.size.width, height: self.walletHeaderSectionHeight)
+        label.leftPadding = 17
+        label.topPadding = 16
+        label.borderWidth = 0
+        label.text = self.wallets[section].name
+        label.textColor = UIColor.appColors.darkBluishGray
+        label.backgroundColor = UIColor.white
+        label.adjustsFontSizeToFitWidth = false
+        label.lineBreakMode = .byTruncatingTail
+        label.numberOfLines = 1
+        label.font = UIFont(name: "SourceSansPro-Regular", size: 14)
+        return label
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.wallets[section].accounts.count
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return self.accountCellRowHeight
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let accountViewCell = tableView.dequeueReusableCell(withIdentifier: "AccountSelectorTableViewCell") as! AccountSelectorTableViewCell
+        let wallet = self.wallets[indexPath.section]
+        let account = wallet.accounts[indexPath.row]
+
+        accountViewCell.account = account
+        accountViewCell.checkmarkImageView.isHidden = !(wallet.name == self.selectedWallet?.name &&
+                                                      account.name == self.selectedAccount?.name)
```

Comment:
Instead of inverting the condition, can do `wallet.name != self.selectedWallet?.name || account.name != self.selectedAccount?.name`. Either works though, no strong preference.